Blueprints/BlueprintDefinitions/vs2022/AnnotationsServerless-Image/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -0,0 +1,154 @@
+is done by using the `--verbosity` switch.
+```
+dotnet build --verbosity detailed
+```
+To change the verbosity in Visual Studio go to Tools -> Options -> Projects and Solutions and adjust the MSBuild verbosity drop down boxes.
+
+## Dependency Injection
+
+Lambda Annotations supports using dependency injection. A class can be marked with a `LambdaStartup` attribute. The class will 
+have a `ConfigureServices` method for configuring services.
+
+The services can be injected by either constructor injection or using the `FromServices` attribute on a method parameter of
+the function decorated with the `LambdaFunction` attribute.
+
+Services injected via the constructor have a lifecycle for the length of the Lambda compute container. For each Lambda 
+invocation a scope is created and the services injected using the `FromServices` attribute are created within the scope.
+
+Example startup class:
+```csharp
+[LambdaStartup]
+public class Startup
+{
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddAWSService<Amazon.S3.IAmazonS3>();
+        services.AddScoped<ITracker, DefaultTracker>();
+    }
+}
+```
+
+Example function using DI:
+```csharp
+public class Functions
+{
+    IAmazonS3 _s3Client;
+
+    public Functions(IAmazonS3 s3Client)
+    {
+        _s3Client = s3Client;
+    }
+
+
+    [LambdaFunction]
+    [HttpApi(HttpMethod.Put, HttpApiVersion.V2, "/process/{name}")]
+    public async Task Process([FromServices] ITracker tracker, string name, [FromBody] string data)
+    {
+        tracker.Record();
+
+        await _s3Client.PutObjectAsync(new PutObjectRequest
+        {
+            BucketName = "storage-bucket",
+            Key = name,
+            ContentBody = data
+        });
+    }
+}
+```
+
+## Lambda .NET Attributes
+
+List of .NET attributes currently supported.
+
+
+* LambdaFunction
+    * Placed on a method. Indicates this method should be exposed as a Lambda function.
+* LambdaStartup
+    * Placed on a class. Indicates this type should be used as the startup class and is used to configure the dependency injection and middleware. There can only be one class in a Lambda project with this attribute.
+
+### Event Attributes    
+
+Event attributes configuring the source generator for the type of event to expect and setup the event source in the CloudFormation template. If an event attribute is not set the
+parameter to the `LambdaFunction` must be the event object and the event source must be configured outside of the code.
+
+* RestApi
+    * Configures the Lambda function to be called from an API Gateway REST API. The HTTP method and resource path are required to be set on the attribute.
+* HttpApi
+    * Configures the Lambda function to be called from an API Gateway HTTP API. The HTTP method, HTTP API payload version and resource path are required to be set on the attribute.
+
+### Parameter Attributes
+
+* FromHeader
+    * Map method parameter to HTTP header value
```

Comment:
nit: ending `.` missing on 4 out of 5 descriptions.

Reply:
Fixed